run.sh

```diff
@@ -23,7 +23,7 @@ for path in $INPUT_PATH; do
 	then
 		echo "🐛 The given path does not exist: $path"
 		echo "🧰 To resolve this issue, set the 'path' parameter to the location of your XML test report(s)."
-		exit 1
+		exit 0
```

Comment:
If I'm understanding the motivation correctly, you're hoping to gracefully handle the scenario where someone specifies _multiple_ paths and some of those paths don't exist (as seen [here](https://github.com/acryldata/datahub/blob/c71a134cce601814c2eb499152924c9ba8eee27d/.github/workflows/metadata-ingestion.yml#L78-L81)).

But, there's no official support for specifying multiple paths. The `path` input is expected to contain exactly one path:

https://github.com/buildpulse/buildpulse-action/blob/c9db3eeef410e5e6c468a21ef2f4585b27c26f7e/README.md#L41-L43

As seen in the [error message in this build](https://github.com/acryldata/datahub/actions/runs/3748034870/jobs/6364916925#step:9:61), the action terminated after discovering that the first path was nonexistent. It sounds like the user wants the action to continue   on to process the remaining paths. If that is indeed the case, then changing the exit code here won't satisfy that goal. Instead, we'd need to enhance this action to explicitly support multiple paths.

To do that, we have a few options:

- Option 1: Iterate over the list of paths specified in the `path` input. For each path that exists on disk, invoke the buildpulse-test-reporter CLI with that path.
- Option 2: Copy all of the paths into a new shared location (e.g., `/tmp/buildpulse-uploadables`) and then invoke the buildpulse-test-reporter CLI with that single path.
- Option 3: Enhance the buildpulse-test-reporter to accept a list of paths instead of a [single path](https://github.com/buildpulse/test-reporter/blob/v0.24.1/cmd/test-reporter/main.go#L25).